### DISCUSSION TITLE
How to update runc to version 1.1.12  on SLES15 to remediate security vulnerability.

### DISCUSSION THREAD

**Discussion by cjmercad72** [opencontainers/runc#4212] [General]
my current  runc version running on my SLES15  environment:
![2024-03-04_07-59-37](https://github.com/opencontainers/runc/assets/101137461/23922420-15c2-4a9a-af34-916d5e5e3e76)

thanks for your time.

**Comment by cyphar** [comment]
If you have a SLE bug, please report it to SUSE directly.

We (SUSE) release the newest runc version in SLES very soon after the upstream version so it's very strange that you still have a version from a year ago. A `zypper ref; zypper up` should be enough unless you've disabled the containers module or something similar.

An updated runc 1.1.11 with the relevant CVE-2024-21626 security patches applied was released in the Containers module on 2024-02-01, and the runc 1.1.12 update was shipped about two weeks later. [See this Bugzilla for the timeline.](https://bugzilla.suse.com/show_bug.cgi?id=CVE-2024-21626)
